### DISCUSSION TITLE
Render font in Italic and/or bold

### DISCUSSION THREAD

**Discussion by scpolo** [nothings/stb#1367] [Q&A] [answered]
Hi all,

We are wondering if it is possible render a text using stb_truetype in italic and/or bold. I mean with a font with a no-bold and no-italic properties.

Any idea?

Thanks and best regards.

**Comment by thedmd** [comment, marked as answer]
Out of the box, this is not supported.

You can try to get glyph outline via `stbtt_GetGlyphShape` and slant them or extrude, then render manually with `stbtt_Rasterize`.
Manipulating outline may not be trivial. Not to mention you will need to patch glyph metrics accordingly. Things will get hairy quickly.

FreeType does support both operations, which may be the best way to go.
